LoopKit/Service/RemoteDataService.swift

```diff
@@ -104,11 +104,13 @@ public protocol RemoteDataService: Service {
     func uploadSettingsData(_ stored: [StoredSettings], completion: @escaping (_ result: Result<Bool, Error>) -> Void)
     
     /**
-     Fetch a RemoteCommand from a push notification
+     Handle a push notification
      - Parameter notification: The push notification dictionary
-     - Returns: RemoteCommand
      */
-    func commandFromPushNotification(_ notification: [String: AnyObject]) async throws -> RemoteCommand
+    func handleRemoteNotification(_ notification: [String: AnyObject]) async throws
+    
```

Comment:
REMOTE 2.0: WILL REMOVE PRE-MERGE

Reply:
nit: "handle" is one of those non-descriptive verbs, like "process".  Every function "handles" something or "processes" something.  Maybe something like `remoteNotificationWasReceived(_ notification ...)`

Reply:
Done